pylot/simulation/perfect_tracker_operator.py

```diff
@@ -51,9 +51,9 @@ def on_watermark(self, timestamp, ground_tracking_stream):
         # Only consider obstacles which still exist at the most recent
         # timestamp.
         for obstacle in obstacles_msg.obstacles:
-            if obstacle.id == self._vehicle_id:
-                # Do no track the ego-vehicle.
-                continue
+            # if obstacle.id == self._vehicle_id:
```

Comment:
Can you please use self._flags.prediction_type != 'r2p2' to control if the ego vehicle track should be included or not?